packages/dataviews/src/components/dataviews-filters/index.tsx

```diff
@@ -28,7 +28,7 @@ export function useFilters( fields: NormalizedField< any >[], view: View ) {
 	return useMemo( () => {
 		const filters: NormalizedFilter[] = [];
 		fields.forEach( ( field ) => {
-			if ( ! field.elements?.length ) {
+			if ( ! field.elements?.length && field.type !== 'integer' ) {
```

Comment:
Why are we hard coding the type here?

Reply:
Because this PR only adds support for the integer type, and from my understanding, we will iterate on this to add support for other types, as @oandregal [commented](https://github.com/Automattic/wp-calypso/pull/103276#discussion_r2084875336). He is also working on another PR to add support for datetime types.

Reply:
For me, this should test the presence of an "Edit" function in the "type" instead of actually hardcoding types.